README.md

```diff
@@ -1,283 +1,487 @@
-
-**dialects**
-* [`coffee`](https://github.com/webpack/coffee-loader): Loads coffee-script like JavaScript
-* [`babel`](https://github.com/babel/babel-loader): Turn ES6 code into vanilla ES5 using [Babel](https://github.com/babel/babel).
-* [`livescript`](https://github.com/appedemic/livescript-loader): Loads LiveScript like JavaScript
-* [`sweetjs`](https://github.com/jlongster/sweetjs-loader): Use sweetjs macros.
-* [`traceur`](https://github.com/jupl/traceur-loader): Use future JavaScript features with [Traceur](https://github.com/google/traceur-compiler).
-* [`typescript`](https://github.com/andreypopp/typescript-loader): Loads TypeScript like JavaScript.
-
-**templating**
-* [`html`](https://github.com/webpack/html-loader): Exports HTML as string, require references to static resources.
-* [`jade`](https://github.com/webpack/jade-loader): Loads jade template and returns a function
-* [`handlebars`](https://github.com/altano/handlebars-loader): Loads handlebars template and returns a function
-* [`ractive`](https://github.com/rstacruz/ractive-loader): Pre-compiles Ractive templates for interactive DOM manipulation
-* [`markdown`](https://github.com/peerigon/markdown-loader): Compiles Markdown to HTML
-* [`ng-cache`](https://github.com/teux/ng-cache-loader): Puts HTML partials in the Angular's $templateCache
-
-**styling**
-* [`style`](https://github.com/webpack/style-loader): Add exports of a module as style to DOM
-* [`css`](https://github.com/webpack/css-loader): Loads css file with resolved imports and returns css code
-* [`postcss`](https://github.com/postcss/postcss-loader): Loads and compiles css using various [plugins](https://github.com/postcss/postcss#plugins)
-* [`less`](https://github.com/webpack/less-loader): Loads and compiles a less file
-* [`sass`](https://github.com/jtangelder/sass-loader): Loads and compiles a scss file
-* [`stylus`](https://github.com/shama/stylus-loader): Loads and compiles a stylus file
-
-**misc**
-* [`po`](https://github.com/perchlayer/po-loader): Loads a PO gettext file and returns JSON
-* [`mocha`](https://github.com/webpack/mocha-loader): Do tests with mocha in browser or node.js
-* [`eslint`](https://github.com/MoOx/eslint-loader): PreLoader for linting code using ESLint.
-* [`jshint`](https://github.com/webpack/jshint-loader): PreLoader for linting code.
-* [`jscs`](https://github.com/unindented/jscs-loader): PreLoader for style checking.
-* [`injectable`](https://github.com/jauco/webpack-injectable): Allow to inject dependencies into modules
-* [`transform`](https://github.com/webpack/transform-loader): Use browserify transforms as loader.
-* [`jsbeautify`](https://github.com/tomaszczechowski/jsbeautify-loader): Autoformatting code.
-
-For the full list of loaders, see [list of loaders](https://webpack.github.io/docs/list-of-loaders.html).
-
-## Module Format (AMD/CommonJS)
-
-webpack supports **both** AMD and CommonJS module styles. It performs clever static
+#### Files
```

Comment:
I also agree, but lets keep them for now since we aren't done with setting it up on webpack.js.org yet.